doc/meson.build

```diff
@@ -1,6 +1,11 @@
 install_man(['mpd.1', 'mpd.conf.5'])
 
 sphinx = find_program('sphinx-build')
+py = import('python').find_installation()
```

Comment:
This doesn't check if this is the Python version used by Sphinx, i.e. this is fragile.